README.md

```diff
@@ -388,14 +384,12 @@ limitations under the License.
   * The managability team at Juniper Networks.
 
 
-[img-pyangbind]: http://rob.sh/img/pyblogo_gh.png
 [img-travis]: https://img.shields.io/travis/robshakir/pyangbind.svg
 [img-codecov]: https://img.shields.io/codecov/c/github/robshakir/pyangbind.svg
 [img-pypi]: https://img.shields.io/pypi/v/pyangbind.svg
 [img-license]: https://img.shields.io/pypi/l/pyangbind.svg
 [img-pyversion]: https://img.shields.io/pypi/pyversions/pyangbind.svg
 
-[pyangbind-docs]: http://pynms.io/pyangbind/
```

Comment:
These docs are the same as the ones in `docs/` if we just want to update the link to there.

Reply:
Done, link updated to /docs